client/main.lua

```diff
@@ -527,7 +527,7 @@ CreateThread(function()
                             QBCore.Functions.Notify(Lang:t("error.not_correct_type"), "error", 3500)
                         end
                     end
-                elseif InputOut then
+                elseif InputOut and currentGarage then
```

Comment:
Maybe add an client console error or an error notification to help debugging !

Like:
```lua
TriggerEvent("QBCore:Notify", "You're not in a garage!", "error")
```

or

```lua
print("current garage is not selected!")
```